core/controllers/acl_decorators.py

```diff
@@ -2992,7 +2992,9 @@ def test_can_update_suggestion(
 
         if suggestion.author_id == self.user_id:
             raise base.UserFacingExceptions.UnauthorizedUserException(
-                'You are not allowed to update suggestions that you created.')
+                'The user %s is not allowed to update self-created'
```

Comment:
Why are you changing this?

Reply:
I changed this in order to identify the user email, in case if flake 1 would still occur in future. In frontend, I have already limited editing self-created suggestions([translations](https://github.com/oppia/oppia/blob/c1fbb7aefcdeab93a393df60a07a0da4a3f45096/core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts#L118)).

Reply:
Then you should use the username; user email is sensitive data.

Reply:
Done